src/channel/bolt/policy.rs

```diff
@@ -220,17 +220,69 @@ impl Default for Policy {
 impl Policy {
     /// Sets policy to match default policy used in c-lightning
     pub fn with_clightning_defaults() -> Policy {
-        todo!()
+        Policy {
+            to_self_delay_max: 14 * 24 * 6,
+            // I  don't know what is the normal operational range for c-lightning
+            // but I only see values way more than 100 in examples like this
+            // https://github.com/ElementsProject/lightning/blob/335ef3fb69f923f6f30213f68881d10ee6d977ed/doc/PLUGINS.md#openchannel
+            // or tests like this https://github.com/ElementsProject/lightning/blob/master/tests/test_wallet.py#L594
+            //feerate_per_kw_range: 1..100,
```

Comment:
I expect that without this field the code will fail to compile. Can you put here something like `1..1000`?

Reply:
done